exercises/leap/canonical-data.json

```diff
@@ -1,39 +1,29 @@
 {
    "cases": [
       {
-         "description": "leap year in twentieth century",
-         "input": 1996,
-         "expected": true
-      },
-      {
-         "description": "odd standard year in twentieth century",
-         "input": 1997,
-         "expected": false
-      },
-      {
-         "description": "even standard year in twentieth century",
-         "input": 1998,
+         "description": "year not divisible by 4: standard year",
+         "input": 2015,
          "expected": false
       },
       {
-         "description": "standard year in nineteenth century",
-         "input": 1900,
-         "expected": false
+         "description": "year divisible by 4, not divisible by 100: leap year",
```

Comment:
I also appreciate the effort you've put into this and hope my differing opinions have not discouraged you.